extra/1-radio-stations.js

```diff
@@ -14,7 +14,13 @@
  */
 
 // `getAllFrequencies` goes here
-
+function getAllFrequencies() {
+  let frequencies = [];
```

Comment:
Well done 👍 

If you're free, you can try to find a different way to do this, without using loop